sdk/communication/azure-communication-common/build.gradle

```diff
@@ -13,6 +13,7 @@ android {
 dependencies {
     implementation "com.fasterxml.jackson.core:jackson-databind:$jacksonVersion"
     implementation "com.jakewharton.threetenabp:threetenabp:$threeTenAbpVersion"
+    implementation project(path: ':sdk:core:azure-core')
```

Comment:
I thought we were trying to remove the dependency on core? I remember removing this in a previous PR I made in Android

Reply:
Right. My Bad